apps/smart-contracts/core/test/Collateral.test.ts

```diff
@@ -941,6 +943,102 @@ describe('=> Collateral', () => {
     })
   })
 
+  describe('# setName', () => {
+    const nonEmptyString = 'Not Empty'
+    it('reverts if not owner', async () => {
+      expect(await collateral.owner()).to.not.eq(user.address)
+
+      await expect(collateral.connect(user).setName(nonEmptyString)).to.be.revertedWith(
+        revertReason('Ownable: caller is not the owner')
+      )
+    })
+
+    it('sets to a non-empty string', async () => {
```

Comment:
```suggestion
    it('sets to non-empty string', async () => {
```
same for test names below